major_event_log/templates/major-event-log/index.html

```diff
@@ -6,28 +6,71 @@
 	<div class="page-header">
 		<h1>Major Event Log</h1>
 	</div>
-	<table class="table table-striped table-hover">
-		<thead>
-			<tr>
-				<th>Event</th>
-				<th>Event Date</th>
-				<th>ID</th>
-				<th>Outcome</th>
-			</tr>
-		</thead>
-		<tbody>
-		{% for event in events %}
-			<tr>
-				<td><a href="{% url 'major-event-log:event_details' event.id %}">{{ event.title }}</a></td>
-				<td>{{ event.date }}</td>
-				<td>{{ event.id }}</td>
-				{% if event.get_outcome_display == 'Success' %}
-				<td><span class="label label-success">{{ event.get_outcome_display }}</span></td>
-				{% else %}
-				<td><span class="label label-danger">{{ event.get_outcome_display }}</span></td>
-				{% endif %}
-			</tr>
-		{% endfor %}
-		</tbody>
-	</table>
+    {% if events %}
+        <table class="table table-striped table-hover">
+            <thead>
+                <tr>
+                    <th>ID</th>
+                    <th>Event Date</th>
+                    <th>Event</th>
+                    <th>Outcome</th>
+                </tr>
+            </thead>
+            <tbody>
+            {% for event in events %}
+                <tr>
+                    <td class="monospaced"><a href="{% url 'major-event-log:event_details' event.id %}">{{ event.id|truncatechars:16 }}</a></td>
+                    <td>{{ event.date }}</td>
+                    <td>{{ event.title }}</td>
+                    {% if event.get_outcome_display == 'Success' %}
+                    <td class="monospaced"><span class="label label-success">{{ event.get_outcome_display }}</span></td>
+                    {% else %}
+                    <td class="monospaced"><span class="label label-danger">{{ event.get_outcome_display }}</span></td>
+                    {% endif %}
```

Comment:
We can strip this if block down to a single element with

```html
<td>
    <span class="label {% if event.get_outcome_display == 'Success' %}label-success{% else %}label-danger{% endif %}>...</span>
</td>
```
And if you really wanted to keep it DRY you could do

```html
<td>
    <span class="label label-{% if event.get_outcome_display == 'Success' %}success{% else %}danger{% endif %}>...</span>
</td>
```

Probably want to open up a new issue to add a `is_success` method to the Event model so that we don't need to  do this
```html
{% if even.get_outcome_display == 'Success' %}
```